packages/desktop-client/src/components/budget/index.tsx

```diff
@@ -297,18 +297,21 @@ function BudgetInner(props: BudgetProps) {
     dispatch(applyBudgetAction(month, type, args));
   };
 
-  const onShowActivity = (categoryName, categoryId, month) => {
+  const onShowActivity = (categoryId, month) => {
+    const conditions = [
+      { field: 'category', op: 'is', value: categoryId, type: 'id' },
+      {
+        field: 'date',
+        op: 'is',
+        value: month,
+        options: { month: true },
+        type: 'date',
+      },
+    ];
     navigate('/accounts', {
       state: {
         goBack: true,
-        filterName: `${categoryName} (${monthUtils.format(
-          month,
-          'MMMM yyyy',
-        )})`,
-        filter: {
```

Comment:
Some parts of the code still depends on this filter object e.g. auto selection of category when adding a new transaction on the category transactions page. Those will break if we remove this.

Reply:
We should either cleanup the usages of the filter state in the Account component or update the code to make the existing feature work with the conditions state.

Reply:
commited a change to remove the new "conditions" variable and instead integrate the incoming data into the "filter" state. This should fix the issue.